src/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM mcr.microsoft.com/azure-functions/dotnet:3.0 AS base
+WORKDIR /app
+
+FROM mcr.microsoft.com/dotnet/core/runtime-deps:2.2-alpine3.8 AS build
+WORKDIR /src
+COPY ["Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue.csproj", "Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/"]
+RUN dotnet restore "Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue.csproj"
+COPY . .
+WORKDIR "/src/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue"
+RUN dotnet build "Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue.csproj" -c Release -o /app/build
+
+FROM build AS publish
+RUN dotnet publish "Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue.csproj" -c Release -o /app/publish
+
+FROM base AS final
+WORKDIR /app
+COPY --from=publish /app/publish .
+CMD [ "/Arcus.Messaging.Tests.Runtimes.AzureFunction.ServiceBus.Queue" ]
```

Comment:
This is not correct, see https://github.com/tomkerkhove/azure-friday-keda/blob/main/src/AzureFriday.Shipments/Dockerfile#L10-L15 for and example.

I think you can reproduce your issue by building it locally in your commandline (no VS magic)

Reply:
Let me check...

Reply:
Ok, now I'm back at the start, where I got the project running, but it ignores the Azure Function:
https://dev.azure.com/codit/Arcus/_build/results?buildId=52849&view=logs&j=af2770d6-9849-5611-d90b-14595ba0772b&t=b20c33dd-af2d-5d04-7049-688a0f4bc19d

Seems like it's running something else?

Reply:
Did you try to run it outside of VS as a container through the command line?

Reply:
Your issue clearly is the dockerfile. You should use azure function CLI then to generate one

`func init --docker-only` I think

Reply:
Reverted my changes to the default file, which was provided by VS, since the default CLI contains not the necessary copy path commands and I don't know them 😄 .

Reply:
Why do you need the COPY parts?

Reply:
That was defaultly added. I think it's better that you take this Docker-part over from me, as I don't really know what or how this is supposed to be working.